File: clientPython/zeroc-ice-3.7.2/csharp/test/Ice/facets/BI.cs
//
// Copyright (c) ZeroC, Inc. All rights reserved.
//

namespace Ice
{
    namespace facets
    {
        public sealed class BI : Test.BDisp_
        {
            public BI()
            {
            }

            public override string callA(Ice.Current current)
            {
                return "A";
            }

            public override string callB(Ice.Current current)
            {
                return "B";
            }
        }
    }
}

File: clientPython/zeroc-ice-3.7.2/matlab/test/Ice/objects/BI.m
%
% Copyright (c) ZeroC, Inc. All rights reserved.
%

classdef BI < Test.B
    methods
        function ice_preMarshal(obj)
            obj.preMarshalInvoked = true;
        end

        function ice_postUnmarshal(obj)
            obj.postUnmarshalInvoked = true;
        end
    end
end
